scripts/apps/search/MultiImageEdit.ts

```diff
@@ -44,7 +44,7 @@ export function MultiImageEditController(
         const updatedImages = imagesOriginal.map((image) => {
             const existingImage = $scope.images.find(({_id}) => _id === image._id);
 
-            return existingImage != null ? existingImage : {...image, ...$scope.metadata};
+            return existingImage != null ? existingImage : {...image, ...pick($scope.metadata, identity)};
```

Comment:
According to docs, the second argument for `pick` is `[paths] (...(string|string[]))` and you are passing a function. How does this work?

Reply:
yep weird, I think it uses pickBy if you pass a function, undocumented feature. will fix that